boards/x86/arduino_101/arduino_101.yaml

```diff
@@ -7,3 +7,4 @@ toolchain:
   - issm
 supported:
   - usb_device
+  - ble
```

Comment:
I think bluetooth is preferable over ble, we don't really refer to ble in any of our APIs. BLE would suggest this works over single mode only, but mesh defines transports such as GATT Proxy which in theory works over BR/EDR aka. classic or dual-mode as well.